bolt-cli/src/common/mod.rs

```diff
@@ -27,6 +30,23 @@ pub mod hash;
 /// Utilities for working with Consensys' Web3Signer remote keystore.
 pub mod web3signer;
 
+/// Parses an ether value from a string.
+///
+/// The amount can be tagged with a unit, e.g. "1ether".
+///
+/// If the string represents an untagged amount (e.g. "100") then
+/// it is interpreted as wei.
+pub fn parse_ether_value(value: &str) -> Result<U256> {
+    Ok(if value.starts_with("0x") {
+        U256::from_str_radix(value, 16)?
+    } else {
+        DynSolType::coerce_str(&DynSolType::Uint(256), value)?
+            .as_uint()
+            .wrap_err("Could not parse ether value from string")?
+            .0
+    })
+}
```

Comment:
this is taken from `cast`

Reply:
Can you add a reference to the code snippet?